spec/features/user_visits_place_spec.rb

```diff
@@ -24,10 +25,10 @@
 
         click_button "Go"
 
-        assert_equal '/places/turing-school-of-software-design',
+        assert_equal '/places/turing-school-of-software-designChIJFSt998R4bIcR0e3QWNXH9D8',
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.